components/Slider/src/MDCSlider.m

```diff
@@ -35,6 +35,8 @@
 @interface MDCSlider () <MDCThumbTrackDelegate>
 @property(nonnull, nonatomic, strong)
     UIImpactFeedbackGenerator *feedbackGenerator API_AVAILABLE(ios(10.0));
+@property(nonatomic) CGFloat previousValue;
+@property(nonatomic) BOOL firstValueChange;
```

Comment:
My understanding based on the code is that this value is being used to avoid having haptics fire when the slider value is initialized, is that correct?

If so, do we need to reset this property when the value is programmatically reset back to the anchor point?

Reply:
Updated the way this is handled to address this concern. Removed firstValueChange, instead set previousValue to -CGFLOAT_MAX when initialized and whenever setValue is called.